src/docs/product/performance/performance-at-scale/index.mdx

```diff
@@ -70,6 +71,16 @@ _Health check_ type transactions, while important for checking the stability of
 
 Some projects within your organisation are very high volume compared to others and this means some low-volume projects might be "drowned out" if we were to treat them all equally. To avoid this problem, we prioritize retaining transactions from lower volume projects, in order to ensure that you have enough samples at any project scale.
 
+### Low Volume Transactions
+
+Within a project, some transaction types occur more often than others, this is a similar problem to the one discussed above, Low Volume Projects, but it applies to various transaction types within a project. To avoid high volume transaction types drowning out low volume transaction types, we prioritise the retention of transaction samples from low volume transaction types, in order to ensure that you have enough samples from all transaction types.
+
```

Comment:
Ok, I'll try to rephrase. Using endpoints instead of "transaction types" is not correct,  transactions are not necessarily related to endpoints. Transactions are whatever the user wants them to be, and an application may not have any "endpoints" (for example a client app). A client app would create transactions based on user interactions, batches it processes or whatever, using endpoints would confuse users whose app does not have endpoints.